src/allmydata/testing/web.py

```diff
@@ -0,0 +1,225 @@
+import hashlib
```

Comment:
needs a copyright statement and docstring.  an `__all__` might be a good thing too.